### DISCUSSION TITLE
Can anyone confirm the new package name or provide an example  ??

### DISCUSSION THREAD

**Discussion by piyushpathar** [jenssegers/laravel-mongodb#2638] [Q&A] [answered]
how to use it in code after the change? Here's what I have so far:

use Jenssegers\Mongodb\Facades\MongoDB;
try {
    // Start a MongoDB transaction (not a real transaction but for error handling)
    MongoDB::startSession();
    MongoDB::beginTransaction();
} catch {

}

**Comment by GromNaN** [comment, marked as answer]
As far as I can see `Jenssegers\Mongodb\Facades\MongoDB` never existed in any version.

To begin a transaction, you can use the `DB` facade of Eloquent.
```php
/** @var \MongoDB\Laravel\Connection */
$mongodb = DB::connection('mongodb');
$mongodb->beginTransaction();
```

**Reply by piyushpathar** [reply]
Thank You
